Dockerfile.linux.testrunner

```diff
@@ -0,0 +1,7 @@
+# Use this Dockerfile to create a Linux testrunner image
```

Comment:
Thinking about the name, I think the term "test" is confusing.  I think the more abstract `runner` term is better.  This isn't being used for tests.

Reply:
Valid point.  I'll update the comment as - `# Use this Dockerfile to create a Linux runner image`